model_analyzer/config/generate/quick_run_config_generator.py

```diff
@@ -144,31 +144,14 @@ def set_last_results(self, measurements: List[RunConfigMeasurement]):
         ----------
         measurements: List of Measurements from the last run(s)
         """
+        self._print_debug_logs(measurements)
 
         self._coordinate_data.increment_visit_count(self._coordinate_to_measure)
         self._neighborhood.coordinate_data.increment_visit_count(
             coordinate=self._coordinate_to_measure)
 
-        if measurements is not None and measurements[0] is not None:
-            assert len(measurements) == 1
-
-            throughput = measurements[0].get_non_gpu_metric_value(
-                "perf_throughput")
-            avg_latency = measurements[0].get_non_gpu_metric_value(
-                "perf_latency_avg")
-
-            self._neighborhood.coordinate_data.set_measurement(
-                coordinate=self._coordinate_to_measure, measurement=measurements[0])
-            logger.debug(
-                f"Measurement for {self._coordinate_to_measure}: "
-                f"throughput = {throughput}, avg_latency = {avg_latency}"
-            )
-        else:
-            self._neighborhood.coordinate_data.set_measurement(
-                coordinate=self._coordinate_to_measure, measurement=None)
-            logger.debug(
-                f"Measurement for {self._coordinate_to_measure}: None."
-            )
+        self._neighborhood.coordinate_data.set_measurement(
+            coordinate=self._coordinate_to_measure, measurement=measurements[0])
```

Comment:
if measurements is none this will fail. I may have lead you astray in telling you to completely remove the if/else

Reply:
Is there a code in the codebase that passes none to this method? Both model_manager.py and evaluate_config_generator.py passes a list (even when the measurement is none).

Reply:
I think you're right. It is always a list, but the member(s) of the list may be none